common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/auth/AuthService.java

```diff
@@ -39,6 +37,7 @@
 @RequiredArgsConstructor
 public class AuthService
 {
+    private static final String AUTHORIZATION = "Authorization";
```

Comment:
Maybe this constant could be public and reused in `AuthService` from JAMP extension module?

Reply:
fixed